packages/@okta/vuepress-site/.vuepress/public/docs/api/postman/authentication.json

```diff
@@ -1,905 +1,1562 @@
 {
```

Comment:
The diff is pretty impenetrable (partly because of the Postman schema change from v1 to v2.1, partly just because), but the changes boil down to four new entries for the Email factor:

- "Enroll Email Factor"
- "Activate Email Factor"
- "Send Email Challenge (OTP)" (following the naming style used for some other factors)
- "Verify Email Factor"

(no new API endpoints, just Email-specific parameters to existing endpoints)